llvm/lib/Analysis/ValueTracking.cpp

```diff
@@ -1158,7 +1158,6 @@ static void computeKnownBitsFromOperator(const Operator *I,
           Known.makeNonNegative();
       }
 
-      assert(!Known.hasConflict() && "Bits known to be one AND zero?");
```

Comment:
Can you please also drop this one? https://github.com/llvm/llvm-project/blob/5878f2421c3d313758e50389f4854d26c7f4867c/llvm/lib/Analysis/ValueTracking.cpp#L2058 It's also a conflict assertion just written differently.